pages/docs/plugins/grouparoo-postgres.mdx

```diff
@@ -0,0 +1,205 @@
+
+- `column`: (**required**) The name of the column to use for the Property.
+- `aggregationMethod`: (**required**, default: `"exact"`) The type of aggregation method to use when extracting the data. The available options will be added to the generated config file as a comment.
+- `sort`: (**required**, default: `null`), // You can sort the results by another column in this table.
+
+A Postgres Table Source also provides the ability to filter your data via the `filters` option. This is a series of rules that will filter data in the database table to find the appropriate value for each Profile for a given Property.
+
+For example, let's say you had a property called `lifetime_value` which summed all the purchases for a given user. Your Source is a `purchases` table that has a `state` column set to either `successful` or `returned`. You may only want to include `successful` purchases. Your `filters` config might look like this:
+
+```js
+{
+  filters: [{ key: "state", op: "equals", match: "successful" }],
+}
+```
+
+The available operators (`op`) will be added to the generated config file as a comment near the `filters` section.
+
+### Query Source
+
+A Query Source is a more flexible way to build properties. With a Query Source, you can add custom SQL commands to your Properties, which could pull data from one or more tables in your database.
+
+The unique thing about the way a Query Source works is that there aren't any unique options for the Source itself. But the Schedule and Properties look a little different.
+
+#### Query Source Schedule
+
+Like a [Table Source Schedule](#table-source-schedule), a Query Source Schedule is included with the generated config file, commented. It has a couple unique options:
+
+- `query`: A SQL query to return that tells Grouparoo which Profiles to check each time the interval occurs.
+- `propertyId`: The `id` of the Grouparoo Property whose data is returned by `options.query`.
+
+Here's an example:
+
+```js
+{
+  options: {
+    query: "SELECT id FROM users WHERE updated_at >= (NOW() - INTERVAL '2 day')",
+    propertyId: "userId"
+  }
+}
+```
+
+#### Query Source Properties
+
+Query Sources are a little simpler when it comes to Properties. You can generate a Property using the CLI:
+
+    $ grouparoo generate postgres:query:property lifetime_value --parent purchases
+
+This will drop a file at `config/properties/lifetime_value.js`. Edit this file to match your desired configuration.
+
+There is one unique option for Query Source Properties:
+
+- `query`: The query to extract the Property. You can use [mustache variables](https://github.com/janl/mustache.js#variables) to represent the keys of other Properties in the system.
+
+Here's an example that sums the values in the `price` column for rows in which the `user_id` column's value matches the value of the Grouparoo Profile's `userId` field (i.e. `userId` is the `id` for the Property in Grouparoo):
```

Comment:
You can use any of the other Properties you've defied in Grouparoo, and they are available to use within mustache `{{ }}` brackets.